database-seed/mongo-loader.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const path = require('path');
+const fs = require('fs');
+const DATABASE_LINK = process.env.DATABASE_LINK || require('../config.js').DATABASE_LINK;
```

Comment:
How does the process.env work?

Reply:
The short version is `process` is a global variable for node, and `env` is the standard method for  passing down secure keys and tokens